conf/messages.en

```diff
@@ -11,7 +11,7 @@ banner.paragraph1.afterLink = about how you use HMRC services. We use this infor
 banner.acceptAllButton = Accept all cookies
 banner.setCookiePreferencesButton = Set cookie preferences
 
-banner.saveConfirmation.paragraph1.beforeLink = You’ve accepted all cookies. You can 
+banner.saveConfirmation.paragraph1.beforeLink = You’ve accepted all cookies. You can
```

Comment:
Are these end of line spaces needed?

Reply:
There is whitespace in the template, so I don't think so.

Reply:
That's cool, in `accessibility-statement-frontend` I think we have a convention of having whitespace within messages rather than template, but also we use whitespace in templates for markup so it's not something I'm too bothered about.

Reply:
That convention obviously passed me by :-)